style.css

```diff
@@ -0,0 +1,152 @@
+            * {
+                margin: 0;
+                padding: 0;
+                box-sizing: border-box;
+            }
+
+            body {
+                font-family: Arial, sans-serif;
+                background-color: #f0f2f5;
+            }
+
+            #main {
+                width: 100vw;
+                height: 100vh;
+                display: flex;
+                flex-direction: column;
+                align-items: center;
+                text-align: center;
+            
+            }
+
+            .navbar {
+                width: 100%;
+                background-color: #ff4757;
+                color: white;
+                display: flex;
+                justify-content: center;
+                align-items: center;
+                height: 80px;
+                padding: 8px;
+                margin-bottom: 20px;
+            }
+
+            .navHead {
+                font-size: 24px;
+                font-weight: bold;
+            }
+
+            .Products {
+                width: 100%;
+                display: flex;
+                flex-wrap: wrap;
+                justify-content: space-around;
+                gap: 12px;
+                overflow-y: scroll;
+            }
+
+            .Product {
+                background-color: white;
+                border: 1px solid #ddd;
+                border-radius: 10px;
+                padding: 20px;
+                width: 30%;
+                box-shadow: 0 2px 5px rgba(0, 0, 0, 0.1);
+            }
+
+            .productTitle {
+                font-size: 20px;
+                margin-bottom: 10px;
+            }
+
+            .productBody {
+                font-size: 16px;
+                margin-bottom: 10px;
+            }
+
+            .productTag {
+                display: flex;
+                flex-wrap: wrap;
+                gap: 10px;
+                margin-bottom: 10px;
+            }
+
+            .tagTitle {
+                font-size: 14px;
+                font-weight: bold;
+                margin-right: 5px;
+                margin-top: 6px;
+            }
+
+            .tagEle {
+                background-color: #ff4757;
+                color: white;
+                font-size: 14px;
+                padding: 5px 10px;
+                border-radius: 5px;
+            }
+
+            .productViews,
+            .productReactions {
+                font-size: 14px;
+                color: #555;
+            }
+
+            .pagination {
```

Comment:
width should be fixed with min size and overflow shold be scrollable.

Reply:
Done